bee-block/src/block.rs

```diff
@@ -313,7 +313,29 @@ pub mod dto {
 #[allow(missing_docs)]
 pub mod inx {
     use super::*;
-    use crate::error::inx::InxError;
+    use crate::{error::inx::InxError, semantic};
+
+    impl From<semantic::ConflictReason> for inx_bindings::proto::block_metadata::ConflictReason {
```

Comment:
Is this not already implemented in `semantic.rs` ?

Reply:
Those are each other's inverse functions. One is from `proto` to Bee, the other is from Bee to `proto`. We provide both directions to get compiler errors when one of the underlying types changes and the other one has to be adapted.

I will move them all to `semantic.rs` though.